src/Stream.cpp

```diff
@@ -47,6 +47,20 @@ nlohmann::json ConversionPath::status_json() const {
   return Document;
 }
 
+std::string ConversionPath::getKafkaTopicName() const {
+  if (kafka_output) {
+    return kafka_output->topic_name();
+  }
+  return "nullptr";
```

Comment:
is `return "nullptr"` the right thing to do here? could we not throw an exception or log?